components/ADempiere/Form/VAllocation/components/Payments.vue

```diff
@@ -0,0 +1,277 @@
+              prop="id"
+              :width="header.width"
+              :label="header.label"
+            />
+          </el-table>
+        </el-card>
+      </div>
+    </el-card>
+    <el-card class="box-card">
+      <!-- <div slot="header" class="clearfix" style="text-align: center;">
+        <b>
+          {{ 'Detalles de los Pago' }}
+        </b>
+      </div> -->
+      <div id="description-payment" class="description-payment" style="display: flex;">
+        <el-card
+          class="box-card"
+          style="width: 50%;float: left;"
+        >
+          <div>
+            <el-form
+              :inline="true"
+              label-position="left"
+              style="padding: 10px !important;"
+            >
+              <el-col :span="24">
+                <el-form-item
+                  label="Cantidad de Registro:"
+                >
+                  <b>
+                    5
+                  </b>
+                </el-form-item>
+              </el-col>
+              <el-col :span="24">
+                <el-form-item
+                  label="Total Factura:"
+                >
+                  <b>
+                    8.506,23
+                  </b>
+                </el-form-item>
+              </el-col>
+              <el-col :span="24">
+                <el-form-item
+                  label="Convertido:"
+                >
+                  <b>
+                    530,63
+                  </b>
+                </el-form-item>
+              </el-col>
+            </el-form>
+          </div>
+        </el-card>
+        <el-card
+          class="box-card"
+          style="width: 50%;float: right;"
+        >
+          <div>
+            <el-form
+              :inline="true"
+              label-position="left"
+              style="padding: 10px !important;"
+            >
+              <el-col :span="24">
+                <el-form-item
+                  label="Pagado:"
+                >
+                  <b>
+                    350,63
+                  </b>
+                </el-form-item>
+              </el-col>
+              <el-col :span="24">
+                <el-form-item
+                  label="Asignado:"
+                >
+                  <b>
+                    170,00
+                  </b>
+                </el-form-item>
+              </el-col>
+              <el-col :span="24">
+                <el-form-item
+                  label="Monto Abierto:"
+                >
+                  <b>
+                    10,00
+                  </b>
+                </el-form-item>
+              </el-col>
+            </el-form>
+          </div>
+        </el-card>
+      </div>
+    </el-card>
+  </div>
+</template>
+
+<script>
+import { defineComponent, ref } from '@vue/composition-api'
+// import lang from '@/lang'
+// components and mixins
+import Carousel from '@theme/components/ADempiere/Carousel'
+// import formMixin from '@theme/components/ADempiere/Form/formMixin.js'
```

Comment:
remove unused comments

Reply:
Added the translation in the following commit [68211fe](https://github.com/solop-develop/frontend-default-theme/pull/442/commits/68211fe355873ffe49725eb888013312491e1af4)